src/components/Icon/Icon.tsx

```diff
@@ -21,6 +22,29 @@ interface Props extends IconProps {}
 
 export function Icon({source, color, backdrop, accessibilityLabel}: Props) {
   const i18n = useI18n();
+  const unstableMonorail = useMonorail();
+
+  /* eslint-disable babel/camelcase */
+  useEffect(() => {
+    if (unstableMonorail) {
```

Comment:
Internally/privately I think it is okay to just call this `monorail`, as the `monorail` object is stable just not the provider API.